src/java.base/share/classes/java/security/Security.java

```diff
@@ -47,6 +47,9 @@
  * implementation-specific location, which is typically the properties file
  * {@code conf/security/java.security} in the Java installation directory.
  *
+ * @implNote If this property file fails to load, the JDK implementation will
+ * throw an unspecified error when initializing the {@code Security} class.
+ *
```

Comment:
I think I'd probably "the property file" rather than "this property file" here but otherwise the implNote looks okay to me.

Reply:
Change "property file" to "properties file".

Reply:
> I think I'd probably use "the property file" rather than "this property file" here but otherwise the implNote looks okay to me.

Ok with me.